WordPress/src/main/java/org/wordpress/android/util/RtlUtils.java

```diff
@@ -0,0 +1,25 @@
+package org.wordpress.android.util;
+
+
+import android.content.Context;
+import android.content.res.Configuration;
+import android.os.Build;
+import android.support.v4.view.ViewCompat;
+import android.view.View;
+
+import org.wordpress.android.*;
```

Comment:
Just a tiny nitpick - we can probably remove this and View import :)

Reply:
Hmm, I wondering where did the "...*" import come from. As my class count to use * is set to 9999. Also the `.View` import is not highlighted as unused. 
Nevertheless I've fixed the imports. Good catch, thanks;).